src/main/java/com/company/TAM2/Main.java

```diff
@@ -0,0 +1,57 @@
+package com.company.TAM2;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main{
+
+    public static final int COLUMNS_COUNT = 2;
+    public static final String COMA = ",";
+    public static final int ACTION_INDEX = 0;
+    public static final int COUNT_INDEX = 1;
+
+    public static void main(String[] args) {
+        String fromFileName =  "orange.csv";
+        String toFileName = "result.txt";
+        getStatistic(fromFileName, toFileName);
+    }
+
+    public static void getStatistic(String fromFileName, String toFileName) {
+        try (
+            BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+            BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName));
```

Comment:
We need to have separate methods
- readFromFile() that reads data from file
- writeToFile() that writes the result report to the file.
and block `try catch` will be located inside of these methods, not inside the getStatistic() method